ruby_on_rails/vcr.md

```diff
@@ -1,8 +1,56 @@
 # VCR
 
-VCR is a gem for recording HTTP requests to external services and replaying them. A VCR setup should be very specifically tied to
-the tests which need VCR because it's quite expensive.
+VCR is a gem for recording HTTP requests to external services and replaying them.
+A VCR setup should be very specifically tied to the tests which need VCR because it's quite expensive.
 
-Here's an article on about some tweaks which can be made to a VCR setup: https://blog.arkency.com/3-tips-to-tune-your-vcr-in-tests/
+Here's an example configuration we use at Renuo:
+
+```rb
+# spec/rails_helper.rb
+
+RSpec.configure do |config|
+  # …
+
+  # Disable VCR completely for tests that are not tagged with :vcr
```

Comment:
This is the main point that I am not sure about, why don't you want to be warned that a remote call is executed? Isn't there a risk that your tests depend on external services?

Reply:
No, your assumption is wrong. _WebMock_ catches everything. Also if your test is not marked with `:vcr`.